src/main/java/racinggame/domain/Cars.java

```diff
@@ -0,0 +1,73 @@
+package racinggame.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import racinggame.dto.CarDto;
+import racinggame.dto.History;
+import racinggame.random.CapacityGenerator;
+
+public class Cars {
+
+    private final SnapShotStore snapShotStore = new SnapShotStore();
+    private List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        if (cars.isEmpty()) {
+            throw new IllegalArgumentException("자동차가 비여 있을 수 없습니다.");
+        }
+        this.cars = cars;
+    }
+
+    public static Cars from(int count) {
+        return new Cars(carInitialization(count));
+    }
+
+    private static ArrayList<Car> carInitialization(int carCount) {
+        ArrayList<Car> cars = new ArrayList<>();
+        for (int i = 0; i < carCount; i++) {
+            cars.add(Car.defaultCar());
+        }
+        return cars;
+    }
```

Comment:
```suggestion
    private static List<Car> carInitialization(int carCount) {
        List<Car> cars = new ArrayList<>();
        for (int i = 0; i < carCount; i++) {
            cars.add(Car.defaultCar());
        }
        return cars;
    }
```


다형성을 위해 인터페이스 타입으로 선언하는 습관을 들이시면 좋을 것 같아요